cli/src/cli.rs

```diff
@@ -73,6 +78,17 @@ pub struct ValidationWorkerCommand {
 	pub socket_path: String,
 }
 
+#[allow(missing_docs)]
+#[derive(Debug, StructOpt)]
+pub struct HostPerfCheckCommand {
+	/// The path for caching the successful result of performance check.
+	#[structopt(long, parse(from_os_str))]
+	pub cache_path: Option<PathBuf>,
```

Comment:
While I still don't understand why we cache this, I would say that we should call this base-path to make it similar to the rest?

Reply:
Caching is for the case of running the check automatically on the first validator start. This is a consideration from the issue description.

Reply:
> --base-path isn't as explanatory as --cache-path since it doesn't reflect the purpose imo

While I see your point, consistency across all commands is better/more important IMO.